speaker/bgp_controller.go

```diff
@@ -60,6 +59,15 @@ type peer struct {
 	session bgp.Session
 }
 
+func (p peer) id() string {
+	ret := ""
```

Comment:
can this be `ret := p.cfg.Iface`, if the addr is not nil then `ret = p.cfg.Addr.String()` and `return ret`?

Reply:
done